Mobile v2.12 changelog (#6824)
mattermost/docs: source/deploy/mobile-app-changelog.md

 - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
 - iPhone 5s devices and later with iOS 12.4+ is required.

### Improvements
 - Calls: Incoming call notifications now appear on the channel list screen.
 - Calls: Added the server name to the call notification banner for a better user experience.
 - Calls: Added support for multiple client sessions for the same user in the same call.
 - Added a new option to the **Channel Info** to be able to auto-follow all threads.
 - Added an option to convert Group Messages to private channels.
 - Added a ``channelType`` parameter to ``persistentNotificationsConfirmation``.
 - Added a new feature to highlight keywords without triggering a notification. This cannot be configured in the mobile app.

### Bug Fixes

>  - Calls: Fixed an issue where the in-app "push" notification overlay appeared when the user already received the incoming call notification banner.

Reviewer: ```suggestion
 - Calls: Fixed an issue where the in-app "push" notification overlay appeared after the user had already received the incoming call notification banner.
```

>  - Fixed an issue where mentions did not appear in the global **Thread** view.

Reviewer: ```suggestion
 - Fixed an issue where mentions did not appear in the global **Threads** view.
```

 - Fixed an issue where an error "Cannot read property 'centerChannelColor' of undefined" appeared.
 - Fixed an issue with opening the same modal twice.
 - Fixed "Invariant Violation: "Home" has not been registered" error when opening the app.

>  - Fixed an issue where toggling of emojis in the emoji picker and quick reactions was not enabled.

Reviewer: ```suggestion
 - Fixed an issue where toggling emojis in the emoji picker and quick reactions were not enabled.
```
